client/post-editor/editor-notice/index.jsx

```diff
@@ -292,6 +292,21 @@ export class EditorNotice extends Component {
 		const { siteId, message, status, onDismissClick } = this.props;
 		const text = this.getErrorMessage() || this.getText( message );
 
+		if ( message === 'updated' || message === 'published' ) {
+			return (
+				<div className={ classNames( 'editor-notice', { 'is-global': true } ) }>
+					{ siteId && <QueryPostTypes siteId={ siteId } /> }
+					{ text && (
+						<Notice
+							{ ...{ status, text, onDismissClick } }
+							duration={ 2000 }
```

Comment:
could you add this as a conditional to the existing block of code?

Reply:
Do you think I should move the parameter to render call or to the parent template?